WordPress/gutenberg
# Unify BlockPreview & BlockPreviewContent into one component

## Description

This PR updates the way `BlockPreview` & `BlockPreviewContent` are used. We want to soon make previews available for plugin developers and these two components were quite confusing together as seen in https://github.com/WordPress/gutenberg/pull/16033#issuecomment-499839090. 

This PR introduces a new version of `BlockPreview` which is used for both scaled and full-size previews (controlled by a prop `isScaled`) and allows you to add custom `className` to the preview wrapper for further styling if needed.

`BlockPreviewContent` is gone. 

The boolean prop `isScaled` is a solution that simulates how the component is currently used in the codebase. It will be soon replaced with different way of doing scaling that is being cooked in #16113. For now, I think it's a good solution that will keep our current usages working the same way while paving the way for the new scaling API and exporting the component for devs to use in plugins.

## How has this been tested?
- in Block Inserter - hover any reusable block. A new "panel" with title "Preview" and the preview itself should show up.
- focus some block with styles (button or quote for example):
  - check the block settings sidebar and expand "Block Styles" panel. It should contain previews
  - open the block switcher and check both thumbnails and also the full preview after hover which should match the styling of the one in block inserter

## Screenshots 

This is `<BlockPreview isScaled blocks={ [ … ] } />` inside Block Styles:

<img width="292" alt="Screenshot 2019-07-31 at 11 47 42" src="https://user-images.githubusercontent.com/156676/62247173-20509c00-b39a-11e9-876f-6d8d49c4bb9f.png">

This is the same preview inside the block switcher. The "panel" on the right was moved from the `BlockPreview` component directly into places where it is being used from (Inserter and Swicther). BlockPreview component is now always **only** the previewed content and nothing else.

<img width="645" alt="Screenshot 2019-07-31 at 13 53 11" src="https://user-images.githubusercontent.com/156676/62247401-96550300-b39a-11e9-8e05-6bbe0af328f4.png">

## Types of changes





## Checklist:
